server/routes/request.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+
+const protect = require('../middleware/auth');
+const { 
+  getAllRequest, 
+  createRequest, 
+  updateRequest, 
+  getRequestById 
+} = require('../controllers/request');
+
+router.param('requestId', getRequestById);

```

Comment:
```suggestion
router.param('/:requestId', getRequestById);

```

Reply:
I am also not sure about this suggestion. As per the express [docs](https://expressjs.com/en/4x/api.html#router.param), we should define params like this. Can you please clarify this me ?